### PR TITLE
Move `count_users` out of roles foreach

### PR DESCRIPTION
**Pull request by frozzare**
This pull request will move `count_users` out of roles foreach since it's expensive to call for every role. See #949 for more information.

This time against develop instead of master.

### CODE REVIEW

File: classes/class-settings.php [hunk 1]
```diff
@@ -817,13 +817,14 @@ public function render_field( $field ) {
 					// Author or Role dropdown menu
 					$author_or_role_values   = array();
 					$author_or_role_selected = array();
+					$users = count_users();
```
Comment (DavidCramer) [inline classes/class-settings.php]: @frozzare You can move this even higher up, outside the `foreach` of the `exclude_row`. Perhaps place it at the beginning of the `rule_list` case.

**Comment by frozzare** [comment]
Done!

File: classes/class-settings.php [hunk 2]
```diff
@@ -771,6 +771,7 @@ public function render_field( $field ) {
 
 				break;
 			case 'rule_list':
+				$users = count_users();
 				$form = new Form_Generator();
 				$output = '<p class="description">' . esc_html( $description ) . '</p>';
```
Comment (DavidCramer) [inline classes/class-settings.php]: Cool. Could you please align the `=` so the build can padd ;)

File: classes/class-settings.php [hunk 3]
```diff
@@ -823,7 +824,7 @@ public function render_field( $field ) {
 							'value' => $role_id,
 							'text' => $role,
 						);
-						$users = count_users();
+
```
Comment (DavidCramer) [inline classes/class-settings.php]: Also, clear up this line ;)

**Pull request merged**